tests/unit/array/storage/elastic/test_del.py

```diff
@@ -0,0 +1,102 @@
+from unittest.mock import MagicMock
```

Comment:
we do not use `unittest` mock, we prefer to use `monkeypatch` from `pytest` since is our own testing framework

Reply:
done